release-notes.txt

```diff
@@ -1,3 +1,775 @@
+  + stm32f4: Add DMA config for SPI
+  + stm32g4: add support
+
+Device Drivers
+--------------
++ atwinc15x0: add WiFi netdev driver
++ at: Add at_urc_isr module to process URCs upon arrival
++ at24cxxx: MTD wrapper for at24cxxx
++ at86rf215:
+  + implement MR-O-QPSK
+  + implement MR-OFDM
++ at86rf2xx: add support for high data rates
++ dfplayer: New driver for the DFPlayer Mini MP3 player
+* ethos: use luid_get_eui48()
++ edbg_eui: add driver to get MAC address from Atmel EDBG
++ mhz19: initial support
++ mrf24j40: add Turbo Mode
++ mtd_spi_eeprom: add MTD wrapper for AT25XXX EEPROMs
++ pca9633: add new driver
++ periph/timer: add timer_set_periodic()
++ periph/i2c: add periph_i2c_reconfigure feature & implementation for sam0
++ periph/rtt: introduce RTT_MIN_OFFSET
++ periph/spi: add periph_spi_reconfigure feature & implementation for sam0
+* pulse_counter: Use C11 atomics & bugfix
++ sdp3x : Add driver for sdp3x sensor
+
+Build System / Tooling
+----------------------
++ make:
+  + add code generator targets
+  + add JLink support for stm32
+  * docker.inc.mk: add and use generic docker_run_make
+  + allow multiple external board dirs
+  + info-global.inc.mk: add info-boards-features-blacklisted
++ riotctrl_shell:
+  + initial import of shell interaction spawn
+  + provide ipv6_nib interactions and parsers
+  + provide netif interactions and parsers
++ tools:
+  + Add script to run radvd on a tun interface
+  + provide tool to parse `pktbuf` output
+  * kconfiglib:
+    * Add riot_kconfig to override default behaviours
+    * kconfiglib: bump to v14.1.0
+
+Testing
+-------
++ Fuzzing framework:
+  + Add AFL-based fuzzing setup for network modules
+  + Add fuzzing application for gcoap
++ ci: setup github actions
++ sys/test_utils/dummy_thread: initial commit
++ tests/irq_disable_restore: Add test for irq_disable() / irq_restore()
++ tests/periph_spi: add benchmark for acquire / release
++ tests/periph_spi_dma: Add test for SPI with DMA
++ uncrustify: add auto uncrustify with blacklist
+
+Kconfig migration
+-----------------
++ armv7_m/cortexm: declare CPU_ARCH and CPU_CORE
++ boards: Model features in Kconfig
+  + arduino-leonardo, arduino-mega2560, atmega128x-based, atmega256rfr2-based,
+    atmega328p-based, cc2538-based, cc26xx_cc13xx-based, efm32-based, esp-based,
+    fe310-based, kinetis-based, msp430-based, samr21-xpro, slwstk6220a
+* kconfig: Declare common feature symbols
++ drivers: Expose Configurations to Kconfig
+  + ads101x, at, at86rf215, bmx055, cc110x, dose.h, fxos8700, hdc1000, isl29020,
+    l3g4200d, lpsxxx, mag3110, mma8x5x, motor_driver, mtd_sdcard, opt3001,
+    pn532, rn2xx3, slipdev, sps30, tcs37727, tmp00x
++ networking: Expose configurations to Kconfig
+  + asymcute, coap, csma
+  + gnrc:
+    + gomach, lwmac, mac, pktdump, pktbuf, rpl, tcp
++ usbus/cdc/ecm: Expose configurations to Kconfig
+
+API Changes
+===========
+- cpu/sam0_common: drop prescaler from timer config
```

Comment:
```suggestion
- cpu/sam0_common: drop prescaler from timer config
- cpu/sam0_common: PWM config now requires GCLK source 
```

Reply:
Which PR was that?

Reply:
Also https://github.com/RIOT-OS/RIOT/pull/14007? Then the label for API change was missing ;-)

Reply:
Clarified offline. It was indeed part of #14007